lib/transaction/transaction.js

```diff
@@ -231,24 +217,37 @@ Transaction.prototype._isFeeDifferent = function() {
     var fee = this._fee;
     var unspent = this._getUnspentValue();
     if (fee !== unspent) {
-      return 'Unspent value is ' + unspent + ' but specified fee is ' + fee;
+      return new errors.Transaction.FeeError.Different(
+          'Unspent value is ' + unspent + ' but specified fee is ' + fee);
```

Comment:
nit: This would be more consistent if the end parenthesis is on the next line and the indentation is only two spaces (like `{` and `}` on functions):
```javascript
return new errors.Transaction.FeeError.Different(
  'Unspent value is ' + unspent + ' but specified fee is ' + fee
);
```

See for example https://github.com/bitpay/bitcore/blob/master/lib/transaction/transaction.js#L363-L365